tanner/api/api.py

```diff
@@ -51,7 +51,7 @@ def __init__(self, redis_client):
             if not query_res:
                 return 'Invalid SNARE UUID'
             for (i, val) in enumerate(query_res):
-                query_res[i] = json.loads(val)
+                query_res[i] = json.loads(val.decode("utf-8"))
```

Comment:
I'd better put encoding things into redis query

Reply:
`zrevrangebyscore` redis query doesn't have encoding parameter. So except for this case, I have added encoding things in the redis query itself. Please look into it.

Reply:
Actually it has this parameter (maybe they forgot to add it into the docs) :)